src/algorithms/truncation/truncationschemes.jl

```diff
@@ -25,3 +41,22 @@ function _TruncationScheme(; alg=Defaults.trscheme, η=nothing)
 
     return isnothing(η) ? alg_type() : alg_type(η)
 end
+
+function truncation_scheme(trscheme::T; kwargs...) where {T<:TensorKit.TruncationScheme}
+    return trscheme
+end
+
+function truncation_scheme(
+    trscheme::SiteDependentTruncation;
+    direction::Int,
+    r::Int,
+    c::Int,
```

Comment:
I think we have used coordinates as positional arguments throughout the rest of the code, so I am leaning slightly more towards that, especially since these aren't optional arguments, and this avoids having to remember that it is `r` instead of `row` and `c` instead of `col` or `column` etc.